### ISSUE TITLE
FR: multi-threading

### ISSUE THREAD

**Issue by Zahrun** [Huluti/Curtail#207]
When using the recursive folder processing, curtail uses 7% of my CPU, that is a single thread. Would be nice to implement a queue so that $(nproc) number of photos can be treated parallelly.

**Comment by ARAKHN1D** [comment]
I think this can be closed now that #255 is merged.

**Comment by Huluti** [comment]
Yes! [Version 1.13.0 is being released](https://github.com/flathub/com.github.huluti.Curtail/pull/21). 
@Zahrun could you do a new test once 1.13.0 will be available (in a couple of hours on flathub)?

**Comment by Zahrun** [comment]
@Huluti yes, I will test it when it comes to flathub

**Comment by Huluti** [comment]
Available!

**Comment by Zahrun** [comment]
@Huluti, it works, thank you, it goes so fast now!

**Comment by Huluti** [comment]
Glad to hear that. Thanks for initial report and for testing the new release!
